### PR TITLE
Added VS Code integration to README.md

### PR DESCRIPTION
**Pull request by Fuzznimp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,6 +67,32 @@ Then add these lines to your package.json:
 
 The React plugin extends the previous Node plugin so it is not needed.
 
+### VS Code
```
Comment (Fenntasy) [inline README.md]: Great, the only thing missing would be to change the little paragraph at the beginning to point to this section (in line 11)

Comment (Fuzznimp) [reply README.md]: You just want me to remove it, right ?

Comment (Fenntasy) [reply README.md]: In a way yes, removing the paragraph but just keep a link like `[VSCode integration](#vs-code)`

Comment (Fuzznimp) [reply README.md]: cd9e9c7
I tend to over use the `inline code`, so let me know if I do 👍

**Comment by Fuzznimp** [comment]
[2576]

**Comment by Fenntasy** [comment]
@Fuzznimp I see you edited the content of your message to add the `ch` part. Unfortunately, as for PR description, clubhouse does not work on updates.

**Comment by Fuzznimp** [comment]
@Fenntasy Damn I was wondering what was wrong ahah. Good catch !

**Comment by Fuzznimp** [comment]
[ch2576]

**Pull request merged**